app/views/recipes/index.html.erb

```diff
@@ -14,3 +14,13 @@
     </div>
   <% end %>
 </div>
+
+
+<%# BOOKMARKS %>
+<%= simple_form_for @favourite_recipe do |f| %>
+  <% if current_user %>
+    <%= f.submit "Bookmark", class: "btn btn-danger pic-large", id: "buttonz" %>
```

Comment:
Can we have a little heart that fills up when marked as favourite like we have on the figma? 🙏

Reply:
a big button on the card will not work